taxi/models.py

```diff
@@ -0,0 +1,25 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+    def __str__(self) -> str:
+        return f"{self.name} ({self.country})"
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=255, unique=True)
+    username = models.CharField(max_length=255, unique=True)
+    email = models.CharField(max_length=255)
+    password = models.CharField(max_length=255)
+    first_name = models.CharField(max_length=255)
+    last_name = models.CharField(max_length=255)
+
```

Comment:
no need to rewrite every field. You can remove 15-19 lines, leave just license_number